### PR TITLE
Make linuxbrew/homebrew install from releases by default

### PR DESCRIPTION
**Pull request by jtattermusch**
-- protobufs should be installed from 3.0.0-alpha-3 release
-- gRPC should be installed from 0.9.0 release

### CODE REVIEW

**Comment by jtattermusch** [comment]
@jcanizales @soltanmm @murgatroid99 @stanley-cheung @tbetbetbe 
I don't think this should break you guys, but adding you to take a quick look anyway.

I tested installing C core on linux, I don't have a mac machine with me right now.

File: Formula/google-protobuf.rb [hunk 1]
```diff
@@ -3,15 +3,9 @@
...
   head "https://github.com/google/protobuf.git"
-  url 'https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.bz2'
-  version "2.6.1"
-  sha1 '6421ee86d8fb4e39f21f56991daa892a3e8d314b'
-
-  devel do
-    url "https://github.com/google/protobuf/archive/v3.0.0-alpha-2.tar.gz"
-    sha256 "46df8649e2a0ce736e37f8f347f92b32a9b8b54d672bf60bd8f6f4d24d283390"
-    version "3.0.0-alpha-2"
-  end
```
Comment (jcanizales) [inline Formula/google-protobuf.rb:14]: Do we want to eventually remove this file and depend on the formula in Homebrew's repo?

Comment (jtattermusch) [inline Formula/google-protobuf.rb:14]: Perhaps, once that one does exactly what we need.

**Comment by jcanizales** [comment]
I too don't think it'll break Objective-C. Will we update to 0.9.1 when it's released next week?

**Comment by jtattermusch** [comment]
Sure, we should update this whenever an official release comes out.

**Comment by tbetbetbe** [comment]
Please update the PR to resolve the conflict

**Comment by jtattermusch** [comment]
Done

**Pull request merged**